yarn.lock

```diff
@@ -20584,7 +20584,7 @@ minimist@1.2.2:
   resolved "https://registry.yarnpkg.com/minimist/-/minimist-1.2.2.tgz#b00a00230a1108c48c169e69a291aafda3aacd63"
   integrity sha512-rIqbOrKb8GJmx/5bc2M0QchhUouMXSpd1RTclXsB41JdL+VtnojfaJR+h7F9k18/4kHUsBFgk80Uk+q569vjPA==
 
-minimist@^1.2.5:
+minimist@^1.2.3, minimist@^1.2.5:
```

Comment:
__optional note:__  I believe you can also switch a bunch of existing `minimist@1.2.0` "references" to `1.2.5` (assuming CI won't object), but up to you:
```
minimist@^1.1.0, minimist@^1.1.1, minimist@^1.1.3, minimist@^1.2.0, minimist@^1.2.3, minimist@^1.2.5, minimist@~1.2.0:
  version "1.2.5"
  resolved "https://registry.yarnpkg.com/minimist/-/minimist-1.2.5.tgz#67d66014b66a6a8aaa0c083c5fd58df4e4e97602"
  integrity sha512-FM9nNUYrRBAELZQT3xeZQ7fmMOBg6nWNmJKTcgsJeaLstP/UODVpGsr5OhXhhXg6f+qtJ8uiZ+PUxkDWcgIXLw==
``` 
And leave `1.2.0` to only those that specifically require it:
```
minimist@1.2.0:
  version "1.2.0"
  resolved "https://registry.yarnpkg.com/minimist/-/minimist-1.2.0.tgz#a35008b20f41383eec1fb914f4cd5df79a264284"
  integrity sha1-o1AIsg9BOD7sH7kU9M1d95omQoQ=
```

Reply:
I have another PR coming up which will fix that 👍

Reply:
Sorry, forgot the link: #60284

Reply:
Once all these individual bump commits have been merged, I'll rebase that one with master and ensure that the `minimist` dependencies are all consolidated